src/app/pages/e-commerce/e-commerce.component.html

```diff
@@ -1,3 +1,19 @@
+<div class="row">
+  <div class="col-12 col-md-6">
+    <ngx-cta-card [title]="'Need a custom solution?'"
+                  [type]="'pantone'"
+                  [link]="'https://www.akveo.com/contacts'"
```

Comment:
let's add `?utm_source=ngx_admin_demo&utm_medium=contact_card` to the and of the link